admin/model_serverless_backup_restore_job.go

```diff
@@ -17,8 +17,8 @@ type ServerlessBackupRestoreJob struct {
 	// Human-readable label that categorizes the restore job to create.
 	DeliveryType string `json:"deliveryType"`
 	// One or more Uniform Resource Locators (URLs) that point to the compressed snapshot files for manual download. MongoDB Cloud returns this parameter when `\"deliveryType\" : \"download\"`.
-	DeliveryUrl      []string       `json:"deliveryUrl,omitempty"`
-	DesiredTimestamp *BSONTimestamp `json:"desiredTimestamp,omitempty"`
+	DeliveryUrl      []string          `json:"deliveryUrl,omitempty"`
+	DesiredTimestamp *ApiBSONTimestamp `json:"desiredTimestamp,omitempty"`
```

Comment:
This name needs to stay. We have it noted as problematic object however having API prefix is an minor inconvinience for rarely used object.